modules/OFConnectionManager2/module/src/cxn_instance.h

```diff
@@ -341,7 +344,17 @@ void ind_cxn_unblock_barrier(connection_t *cxn);
 void ind_cxn_pause(connection_t *cxn);
 void ind_cxn_resume(connection_t *cxn);
 
-void ind_cxn_process_message(connection_t *cxn, of_object_t *obj);
+void ind_cxn_block_async_op(connection_t *cxn);
+void ind_cxn_unblock_async_op(connection_t *cxn);
+
+void ind_cxn_block_iter_async_op(connection_t *cxn);
+void ind_cxn_unblock_iter_async_op(connection_t *cxn);
```

Comment:
I do not see the above 2 functions being defined, do you want to remove these prototypes?